chain/indexer/src/handlers.rs

```diff
@@ -650,6 +650,36 @@ where
         Ok(None)
     }
 
+    // TODO: uncomment this once ticket winning probability oracle is implemented
+    /*async fn on_ticket_winning_probability_oracle_event(
+        &self,
+        tx: &OpenTransaction,
+        event: HoprTicketWinningProbabilityOracleEvents,
+    ) -> Result<Option<ChainEventType>> {
+        #[cfg(all(feature = "prometheus", not(test)))]
+        METRIC_INDEXER_LOG_COUNTERS.increment(&["win_prob_oracle"]);
+
+        match event {
+            HoprTicketWinningProbabilityOracleEvents::TicketMinWinningProbability(update) => {
+                trace!(
+                    "on_ticket_minimum_win_prob_updated - old: {:?} - new: {:?}",
+                    update.0.to_string(),
+                    update.1.to_string()
+                );
+
+                self.db
+                    .set_minimum_incoming_ticket_win_prob(Some(tx), update.1)
+                    .await?;
```

Comment:
There should be a conversion here before storing to the db. From SC, the value obtained is a `uint56` and needs to be converted to float by dividing `0xffffffffffffff`

Reply:
Ok, good to know. I will reflect that once we have the bindings in Rust ready for the SC.